script/create_pghero_user.rb

```diff
@@ -0,0 +1,74 @@
+# Run with:
+# (on local host) ruby script/create_pghero_user.rb
+# (in docker compose) docker compose exec web ruby script/create_pghero_user.rb
+
+require_relative '../config/environment'
+
+require 'uri'
+
+url = ENV.fetch('PGHERO_DATABASE_URL')
+uri = URI.parse(url)
+pghero_password = uri.password
+database_name = uri.path.delete_prefix('/')
+main_db_user = 'david_runger'
+
+ApplicationRecord.connection.execute(<<~SQL)
+  CREATE SCHEMA pghero;
```

Comment:
Tweaked very slightly (mostly just filling in the blanks) from here https://github.com/ankane/pghero/blob/master/guides/Permissions.md .